src/components/app/app.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import 'normalize.css';
+import '../../styles/global.css';
+
+const App = () => (
+  <Router>
+    <Switch>
+      <Route exact path="/">
+        <div>
+          <code>/</code>
+          {' '}
+          placeholder
+        </div>
```

Comment:
Your PR looks good. I like that you wrote a detailed description 🚀 

Just FYI: This code here and the placeholder for the search bar could be shorter just to remove some clutter. You can simply write 

```
<Route exact path="/">
  Home
</Route>
```

Reply:
Hey, thanks!

Yeah, it's strange. I believe ESLint is the culprit for the whitespace insertions in the App component. I ran `yarn format`, which doesn't bother removing the whitespace. I had disabled Prettier (which I have configured globally in VSCode) as well.

In any case, I will opt for your simpler approach with the `Home` placeholder. I also intend to go with your `styled-components` strategy for managing global styles.

I noticed in your linked example that the semi-bold fonts are included, yet in the marketing site mock-up, there appears to only be regular weights. I suspect the Reddit Timer app (that's included in premium) uses them.

Again, thanks for taking a look at everything! I really appreciate it.

Reply:
You're welcome. About the fonts: I'm not sure tbh. Maybe they are used on the search page. Maybe it's a left-over from an earlier implementation. I cleaned up the designs at some point. Maybe I forgot to change the example implementation 😉